db/fixtures/miq_product_features.yml

```diff
@@ -4186,6 +4186,16 @@
       :feature_type: view
       :identifier: cloud_database_flavor_show
 
+# Vm Infra
+- :name: Vm Infras
+  :description: Every thing to Vm infras
+  :feature_type: node
+  :identifier: vm_infra
+  :children:
+  - :name: View
+    :description: View Vm Infras
+    :feature_type: view
+    :identifier: vm_infra_view
```

Comment:
Confused why we're adding a new top level node when we already have `Reconfigure VMs` / `vm_reconfigure_all` https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_product_features.yml#L5949-L5973

Reply:
Do we need infra specific features?
Can we use 1 set of privileges for both views?